### ISSUE TITLE
Decoding DataInterfaces

### ISSUE THREAD

**Issue by felixp8** [catalystneuro/stavisky-lab-to-nwb#5]
DataInterfaces for speech decoding output. Includes:
* "logits" from RNN decoder - waiting on more clarification on what exactly this is
* decoded text - look into `ndx-events`?

**Comment by felixp8** [comment]
Just making a note of things still to be resolved:
- whether timestamps should be re-interpolated to be regularly sampled, or alternatively, aligned with corresponding binned neural data. if the latter, may need to subclass `BaseTemporalAlignmentInterface` and make use of that functionality
- whether end of trial events are initiated by the decoding model and should be stored
